newrelic/resource_newrelic_synthetics_alert_condition.go

```diff
@@ -100,13 +100,17 @@ func resourceNewRelicSyntheticsAlertConditionCreate(ctx context.Context, d *sche
 
 	monitorGUID := d.Get("monitor_id").(string)
 
-	monitorID := getMonitorID(monitorGUID)
+	monitorID, err := getMonitorID(monitorGUID)
+
+	if err != nil {
+		return diag.FromErr(err)
+	}
 
 	condition := expandSyntheticsCondition(d, monitorID)
 
 	log.Printf("[INFO] Creating New Relic Synthetics alert condition %s", condition.Name)
 
-	condition, err := client.Alerts.CreateSyntheticsConditionWithContext(ctx, policyID, *condition)
+	condition, err = client.Alerts.CreateSyntheticsConditionWithContext(ctx, policyID, *condition)
```

Comment:
Can you please use a new variable such as `createSyntheticsConditionResponse` to replace `condition` here